css/theme.css

```diff
@@ -231,27 +242,25 @@ iframe {
 }
 
 .windowctrl {
+  background: url('sprite-glyph.png') -100px 0 no-repeat;
```

Comment:
Thanks for catching this mistake. I had integrated glyphs for the stoplights, but pulled it out. The OSX default glyphs are too visually indistinct. I suspect that we might do better by tweaking the native style just a bit. Need some time to sketch this out.